lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraSceneIndex.cpp

```diff
@@ -71,10 +71,13 @@ TF_DEFINE_ENV_SETTING(MAYA_HYDRA_USE_MESH_ADAPTER, false,
     "Use mesh adapter instead of MRenderItem for Maya meshes.");
 
 bool useMeshAdapter() {
-    static bool uma = TfGetEnvSetting(MAYA_HYDRA_USE_MESH_ADAPTER);
+    static const bool uma = TfGetEnvSetting(MAYA_HYDRA_USE_MESH_ADAPTER);
     return uma;
 }
 
+TF_DEFINE_ENV_SETTING(MAYA_HYDRA_PASS_NORMALS_TO_HYDRA, true,
```

Comment:
With passing normal as true by default, is there any test case to be rebaselined? 
also, it's better to have a test case for normal if there's no one.

Reply:
With the new normals, all the tests should pass without being rebaselined. I haven't added any tests since I think it will be difficult to differentiate between the generated normals and the passed normals, there are only a few details which are different.

Reply:
Is there no way to generate a small synthetic scene (e.g. a mesh cube maybe?) with passed normals that would be completely different from generated normals (e.g. the opposite sign), and look at the normal values in the Hydra scene index scene, i.e. without image comparison?  Don't know if this is valid / realistic, just thinking out loud.

Reply:
You're right, I can use some scene from the bugs we have when not using the normals from OGS. 
I'll add this as a test. Thanks.